src/redux/actions/projects/createProject.js

```diff
@@ -24,8 +25,12 @@ const createProject = (
 
   const newProjectUuid = uuidv4();
 
-  // Always create an experiment for a new project
-  // required because samples DynamoDB require experimentId
+  if (api.CURRENT_VERSION === api.possibleVersions.V2) {
+    await dispatch(createExperiment(newProjectUuid, newExperimentName));
+
+    return;
```

Comment:
Are you returning here because there's no project creation anymore in the v2? Can you add comment about that here?